src/components/forms/Form/useForm.tsx

```diff
@@ -85,7 +86,11 @@ export class CubeFormInstance<
 
       flag = true;
 
-      field.value = newData[name];
+      if (!inputOnly) {
+        field.value = newData[name];
+      }
+
+      field.inputValue = newData[name];
```

Comment:
I think it brings complication in behavior of other components. Like in case we have `<Input />` inside form, then when we get `field` instance of data we got two sources of data `value` and `inputValue` it really looks confusing.

Reply:
Why we need two sources of data? What case it solves? 

Can we just switch behavior by attribute from child component?

What cases I see for `ComboBox` 
1. When it behaves like normal `Input` with ability to suggest some variants.
2. Like `Select` where entered data is restricted by set of suggested variants.

What if we just need switch on witch event we read data `onInputChange`/`inputValue` or `onSelectionChange`/`selectedKey`.
How we can detect on witch? Simply by `child.props.allowsCustomValue`

Reply:
That's exactly what I did. But we still need value to sync.